tests/fixtures/noqa.py

```diff
@@ -582,3 +582,7 @@ def nested_annotation_return_func() -> Union[str, Union[int, float]]: # noqa: WP
 from json import loads  # noqa: WPS347
 from some_module import a  # noqa: WPS347
 from text import from_file  # noqa: WPS347
+
+constant[0:7]  # noqa: WPS348,WPS428
```

Comment:
Let's replace it with `print(constant[0:7])  # noqa: WPS348`

Why? We don't need to test all kind of things here. We only test that `noqa` works fine. So, one example is enough.

Reply:
You mean just testing that `# noqa` works for the `[0:7]` case? Or testing the three cases but add `print` to avoid WPS428?

Reply:
Yes, we need to test that a single `noqa` works for this violation. We also try to avoid unrelated violations. So, a single `print(constant[0:7]) # noqa: WPS348` is enough.